src/components/account-info/account-info-header.tsx

```diff
@@ -6,30 +6,45 @@ import {Color} from '@app/colors';
 import {
   CardSmall,
   CopyButton,
+  First,
   Icon,
   IconButton,
   Inline,
   Spacer,
   Text,
 } from '@app/components/ui';
 import {createTheme} from '@app/helpers';
+import {Feature, isFeatureEnabled} from '@app/helpers/is-feature-enabled';
 import {shortAddress} from '@app/helpers/short-address';
 import {I18N} from '@app/i18n';
 import {Wallet} from '@app/models/wallet';
 import {Balance} from '@app/types';
 
+import {StackedVestedTokens} from '../stacked-vested-tokens';
+
 const CARD_WIDTH = 78;
 const CARD_RADIUS = 8;
 
 export type AccountInfoProps = {
   wallet: Wallet;
-  balance: Balance;
+  balance: Balance | undefined;
```

Comment:
And down to `onSend`
```suggestion
  balance?: Balance;
```